tests/Shared/Aspire.Workload.Testing.targets

```diff
@@ -19,6 +19,9 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Runtime.WorkloadTesting.Internal" />
+
+    <AdditionalSharedFrameworkToInstallArguments Include="-Runtime dotnet -Version 8.0.8 -Channel 8.0" />
```

Comment:
Is there a way to reuse the dotnet installed by the original build.cmd/sh call so we don't have to add this? I want to avoid needing to update a number in several places (runtime framework in this case 8.0.8) as those change each month.

Reply:
This is specifically for workload testing. It will be much cleaner to use the existing support for this. But we could try to use the restored `.dotnet` also, but also need to account for `.dotnet` not being installed when the system `dotnet` is sufficient.